detox/test/package.json

```diff
@@ -50,16 +50,18 @@
     "@babel/core": "^7.20.0",
     "@babel/preset-env": "^7.20.0",
     "@babel/runtime": "^7.20.0",
-    "@react-native/eslint-config": "^0.73.2",
-    "@react-native/metro-config": "^0.73.3",
-    "@react-native/typescript-config": "0.73.1",
+    "@react-native/babel-preset": "0.74.88",
+    "@react-native/eslint-config": "0.74.88",
+    "@react-native/metro-config": "0.74.88",
+    "@react-native/typescript-config": "0.74.88",
     "@tsconfig/react-native": "^3.0.0",
     "@types/jest": "^29.5.11",
     "@types/node": "^16.18.68",
-    "@types/react": "^18.2.45",
+    "@types/react": "^18.2.6",
     "@typescript-eslint/eslint-plugin": "^6.16.0",
     "@typescript-eslint/parser": "^6.16.0",
     "axios": "^1.7.7",
+    "babel-jest": "^29.6.3",
```

Comment:
Is this really needed (my guess is that - no)

Reply:
Copy pasted it from the template.